movo_robot/movo_bringup/launch/manipulation/movo_moveit.launch

```diff
@@ -11,7 +11,9 @@
     <include if="$(optenv MOVO_HAS_KINOVA_ARM_7DOF false)" file="$(find movo_7dof_moveit_config)/launch/movo_moveit_planning_execution.launch"/>
     <include if="$(optenv MOVO_HAS_KINOVA_ARM_6DOF false)" file="$(find movo_moveit_config)/launch/movo_moveit_planning_execution.launch"/>
     
-	<node pkg="movo_bringup" type="init_robot" name="init_robot" respawn="false" output="screen">
+	<!-- Removed arm init since motion is not necessary and impedes box urdf integration-->
```

Comment:
this looks like wrong-formed xml. you have node tag that closes that never opened. dont u get errors when run movo_system?

you should delete lines 14-22 inclusive

Reply:
hm I see, I haven't run this on the robot yet since the sim doesn't run this code directly. If it shouldn't be running then maybe I've got the wrong file edited cause otherwise we would get an error on startup

Reply:
ah my change would have broken things. I'll remove the lines and this should be good to go